ingress-controller/pkg/telemetry/reports.go

```diff
@@ -82,7 +82,7 @@ func SetupAnonymousReports(
 	}
 
 	fixedPayload := types.Payload{
-		"v":  metadata.Release,
+		"v":  metadata.Metadata().Release,
```

Comment:
REVIEW: should we add a field to mark that the message is sent from KO to distinguish between the message sent from existing KIC instances?

Reply:
renamed in 49da05fa2f9d866a31c3bc3319c7e2319267206d.